CMakeLists.txt.user.18

```diff
@@ -1,1627 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE QtCreatorProject>
-<!-- Written by QtCreator 4.2.0, 2017-01-18T10:53:40. -->
```

Comment:
@pold500 Yes, you can squash them in one commit, just describe changes made in mock_message_helper related files and application_manager_impl_test related files in the commit message.